packages/mantine/src/components/table/Table.types.ts

```diff
@@ -76,6 +76,10 @@ export interface TableLayoutProps<T = unknown> {
      * @param datum the row for which the children should be generated.
      */
     getExpandChildren?: (datum: T) => ReactNode;
+    /**
+     * Wether clicking on a selected row will deselect the row or not.
+     */
+    forceRowSelected?: boolean;
```

Comment:
I'm not sure about the name of the props, I think I'd prefer something like `disableRowDeselection`?

Reply:
Alright no problem, changed it. ✅